pyinfra/operations/docker.py

```diff
@@ -0,0 +1,339 @@
+"""
+Manager Docker Containers, Volumes and Networks
+"""
+
+from pyinfra import host
+from pyinfra.api import operation
+from pyinfra.facts.docker import DockerContainers, DockerNetworks, DockerVolumes
+
+from .util.docker import handle_docker
+
+
+@operation()
+def container(
+    container,
+    image="",
+    ports=[],
```

Comment:
Mutable arguments are dangerous in Python, we should default to `None` for these and default create the empty lists within the function body.

Reply:
resolved in commit [0ea1976](https://github.com/pyinfra-dev/pyinfra/pull/1114/commits/0ea1976f4fa2881d636e952a0f2018e891d47434)